### PR TITLE
Ready to review: clean up metabolic epilepsy branch

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
1. biotinidase deficiency:
- update definition
- exclude parent (metabolic epilepsy, epilepsy is a feature of this disease)
2.  'holocarboxylase synthetase deficiency'
- update def
- exclude parent (metabolic epilepsy, epilepsy is a feature of this disease)
3. 'MERRF syndrome'
- update def
- exclude parent (metabolic epilepsy, epilepsy is a feature of this disease)
4. 'mitochondrial DNA depletion syndrome 4a'
- update def
- exclude parent (metabolic epilepsy, epilepsy is a feature of this disease)
5. 'NARP syndrome'
- update def
- exclude parent (metabolic epilepsy, there is no source and epilepsy is not even mentioned as a feature in the def)
6. oxoglutaricaciduria
- exclude parent (metabolic epilepsy, there is no source, epilepsy is a feature)
7. 'folinic acid-responsive seizures' - revise def, add source

...

I made additional edits of the same nature to the metabolic epilepsy branch, see commit messages below. Let me know if you have any questions when you review the branch.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -1922,16 +1922,16 @@ subset: gard_rare {source="MONDO:GARD"}
...
 xref: MEDGEN:337833 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: NANDO:1200799 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: OMIMPS:606777 {source="MONDO:equivalentTo"}
 xref: UMLS:C1847501 {source="MONDO:equivalentTo", source="MEDGEN:337833", source="MONDO:MEDGEN"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
-is_a: MONDO:0100033 {source="https://orcid.org/0000-0001-8486-0558"} ! metabolic epilepsy
+is_a: MONDO:0100033 {source="https://orcid.org/0000-0001-8486-0558", source="https://www.epilepsydiagnosis.org/aetiology/metabolic-groupoverview.html#glucose"} ! metabolic epilepsy
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0000188 {source="MONDO:CLINGEN"}
-relationship: disease_has_basis_in_disruption_of GO:0005355 ! obsolete glucose transmembrane transporter activity
+relationship: disease_has_basis_in_disruption_of GO:0055056 ! D-glucose transmembrane transporter activity
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:1934]: Could you please add a source for this axiom?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -1922,16 +1922,16 @@ subset: gard_rare {source="MONDO:GARD"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "GLUT1 deficiency syndrome" EXACT CLINGEN_LABEL []
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:1924]: can we add a gene association for this term?

**Review by sabrinatoro — COMMENTED** [review]
@nicolevasilevsky I have 2 comments for you in-line.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:1924]: this has the axiom:
'disease has basis in disruption of' some 'D-glucose transmembrane transporter activity'

I don't think we normally add associated genes for gene deficiency syndromes, it seems like we add axioms that describe the disruption of the biological process

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:1934]: I don't know the source, there wasn't one there before. It probably came from Chris.

**Review by sabrinatoro — APPROVED** [review]
We should get back to this later and make sure to add a gene association and we add a source for the axiom. 
I am approving and merging now so we can have these changes done for the workshop.

**Pull request merged**